Sources/Client/Client/ClientLogger.swift

```diff
@@ -299,7 +299,33 @@ public final class ClientLogger {
     public static func log(_ icon: String, dateTime: String = "", _ message: String) {
         ClientLogger.logger(icon, dateTime, message)
     }
-    
+
+    /// Performs `Swift.assert` and stops program execution if `condition` evaluated to false. In RELEASE builds only
+    /// logs the failure.
+    ///
+    /// - Parameters:
+    ///   - condition: The condition to test.
+    ///   - message: A custom message to log if `condition` is evaluated to false.
+    public static func logAssert(
+        _ condition: Bool,
```

Comment:
In our code style, we don't start parameters from a new line. Can you please follow that?

Reply:
fixed in 
b4ba483